twitch_chatbot.py

```diff
@@ -87,7 +90,9 @@ async def skin(ctx):
     if skin:
         # removing leading and trailing whitespaces
         skin_stripped = " ".join(skin['skin'].split())
-        await ctx.send(f"/me Right now {env['CHANNEL']} is using {skin_stripped}")
+        
+        # TODO find a way to shorten the skin url
```

Comment:
maybe [bit.ly(https://dev.bitly.com/)( api? should also add an option in .env to enable the shortener
something like `BITLY_SHORTENER = True`